Makefile

```diff
@@ -389,8 +389,8 @@ devdoc-update:
 ###                                Protobuf                                 ###
 ###############################################################################
 
-protoVer=v0.8
-protoImageName=tendermintdev/sdk-proto-gen:$(protoVer)
+protoVer=0.8
+protoImageName=ghcr.io/cosmos/proto-builder:$(protoVer)
```

Comment:
That's cool! We may need to inform people using this that the image on the hub.docker.com won't be updated anymore. Does it need a changelog, then? Or an upgrading.md entry?